hal/armv7m/cpu.c

```diff
@@ -228,7 +234,7 @@ void _hal_cpuInit(void)
 
 #ifdef CPU_STM32
 	_stm32_platformInit();
-#elif CPU_IMXRT
+#elif defined(CPU_IMXRT)
 	_imxrt_platformInit();
 #endif
 }
```

Comment:
Maybe this functions should be implemented in platform specific file instead?

Reply:
I agree with you. This function should be implemented in platform specific file and it should be named using common name e.g. _platform_init().

Reply:
I agree with you. However I would like to do these changes in the next PR.